modules/ROOT/pages/neo4j-admin-neo4j-cli.adoc

```diff
@@ -155,7 +155,7 @@ For details, see xref:tools/neo4j-admin/neo4j-admin-store-info.adoc[Display stor
 | `upload`
```

Comment:
Add the xref for the `neo4j-admin database migrate` command.
On a new line, after line 150:
For details, see xref:database-administration/standard-databases/migrate-database.adoc[].